root/etc/e-smith/templates/etc/opt/rh/rh-php72/php-fpm.d/000-nextcloud.conf/10base

```diff
@@ -0,0 +1,58 @@
+{
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro() || die("Can't open config db");
+
+    $OUT .= q(
+;
+;  nextcloud php-fpm configuration
+;
+
+[netserver-nextcloud]
+listen = /var/run/rh-php72-php-fpm/nethserver-nextcloud-php72.sock
+
+;Logs
+php_admin_value[error_log] = /var/opt/rh/rh-php72/log/php-fpm/error-nextcloud.log
+php_admin_flag[log_errors] = on
+
+pm = dynamic
+pm.max_children = 50
+pm.start_servers = 5
+pm.min_spare_servers = 5
+pm.max_spare_servers = 35
+
+user = apache;
+group = apache
+listen.owner = root
+listen.group = apache
+listen.mode = 0660
+
+; Set data paths to directories owned by process user
+php_value[session.save_handler] = files
+php_value[session.save_path]    = /var/opt/rh/rh-php72/lib/php/session
+php_value[soap.wsdl_cache_dir]  = /var/opt/rh/rh-php72/lib/php/wsdlcache
+
+; Set opcache settings 
+php_value[opcache.file_cache]  = /var/opt/rh/rh-php72/lib/php/opcache
+php_value[opcache.enable]  = 1
+php_value[opcache.enable_cli]  = 1
+php_value[opcache.interned_strings_buffer]  = 8
+php_value[opcache.max_accelerated_files]  = 10000
+php_value[opcache.memory_consumption]  = 128
+php_value[opcache.save_comments]  = 1
+php_value[opcache.revalidate_freq]  = 1
+
+);
+
+    my $MemoryLimit = $db->get_prop("nextcloud",'MemoryLimit') || '512';
+    my $UpMaxFileSize = $db->get_prop("nextcloud",'UploadMaxFilesize') || '4';
+    my $PostMaxSize = $db->get_prop("nextcloud",'PostMaxSize') || '8';
+    my $MaxExecTime = $db->get_prop("nextcloud",'MaxExecutionTime') || '0';
+    my $MaxFileUploads = $db->get_prop("nextcloud",'MaxFileUploads') || '20';
```

Comment:
We could inherit global PHP TimeZone as well...

Reply:
done